modules/check_in/spec/workers/travel_claim_submission_worker_spec.rb

```diff
@@ -159,5 +164,164 @@
         end
       end
     end
+
+    context 'travel claim returns success with patient cell phone' do
+      before do
+        allow(Flipper).to receive(:enabled?).with('check_in_experience_mock_enabled').and_return(false)
```

Comment:
This is not needed for every spec since it is being done in the upper level `before` block